config/routes.rb

```diff
@@ -4,8 +4,9 @@
 
   resources :users, only: [:index, :show, :create] do
     get '/discover', to: 'users#discover'
-    post '/movies', to: 'users#movies'
-    get '/movies', to: 'users#movies'
+    resources :movies, only: [:index, :show, :create] do
+      get '/viewing-party/new', to: 'parties#new'
```

Comment:
should we use `viewing_party` or `viewing-party`? I don't often see the '-'. Our model is called party or Parties. Should this be changed?